demo/api/src/pages/blocks/PageContentBlock.ts

```diff
@@ -23,6 +30,7 @@ const supportedBlocks = {
     textImage: TextImageBlock,
     damVideo: DamVideoBlock,
     youTubeVideo: YouTubeVideoBlock,
+    vimeoVideo: VimeoVideoBlock,
```

Comment:
In PageContentBlock we should only have the VideoBlock and remove the other video blocks